src/util/text.js

```diff
@@ -24,20 +24,24 @@ export function sanitize (text) {
   })
 }
 
-export function present (text, opts) {
+export function present (text, opts = {}) {
   if (!text) return '<p></p>'
 
   // wrap in a <p> tag
   if (text.substring(0, 3) !== '<p>') text = `<p>${text}</p>`
 
   // make links and hashtags
-  text = linkify(text)
+  text = linkify(text, opts.slug)
 
   if (opts && opts.maxlength) text = truncate(text, opts.maxlength)
 
   return text
 }
 
+export function formatPostTitle (text, slug) {
```

Comment:
this is basically `present` without wrapping in a `<p>` tag -- can you do that instead, i.e. add a `skipWrap` option to `present`, or add a `wrap` option? it feels wrong to have a utility file know about posts